content/08-mongodb/11-mongodb-atlas-setup.mdx

```diff
@@ -0,0 +1,136 @@
+
+<PrismaOutlinks>
+
+If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) with MongoDB, you can use the [MongoDB connector (currently in preview)](https://www.prisma.io/docs/concepts/database-connectors/mongodb) to connect and manage your data.
+
+</PrismaOutlinks>
+
+## How to provision a MongoDB Atlas cluster
+
+To begin setting up your new Atlas cluster, you will navigate to the [Atlas registration page](https://www.mongodb.com/cloud/atlas/register) and create a new account. After successful registration, you will be directed to a page where you will create your first cluster.
+
+If you already have a MongoDB account registered, you can login from the registration page. Once logged in, you will want to click on the green "+ Create" button pictured below. This will take you to the same page for creating a cluster as if you were creating a cluster for the first time.
+
+![Create Cluster](/content/mdb-atlas-setup/create-cluster-button.png)
+
+The first decision you will have to make is what kind of cluster you would like to configure. At the time of this writing, MongoDB offers three options: Serverless (Preview), Dedicated, and Shared. 
+
+![Cluster Type](/content/mdb-atlas-setup/cluster-type.png)
+
+The serverless instance is suitable for serverless applications with variable or infrequent traffic. There is minimal configuration and resources scale to meet your workload with cost matching your actual usage.
+
+The dedicated instance is the selection we will continue walking through for the remainder of the guide. It is for production applications with sophisticated workload requirements. There are advanced configuration controls such as network isolation, end-to-end encryption, and fine-grained access controls. The dedicated cluster also has on-demand performance advice around indexes, data schema, and more.
+
+Lastly, is the shared cluster instance. This cluster type is for learning and exploring MongoDB in a sandbox-like environment. It has basic configuration controls and is best for working with sample datasets and familiarizing yourself with MongoDB.
+
+### Global Cluster Configuration
+
+The first section after selecting a Dedicated cluster instance is Global Cluster Configuration. It displays a map that shows the approximate write latency of every region from high to low. Atlas Global Clusters are for sharded cluzter zones to support location-aware read and write operations for globally distributed applications.
+
+![Global Cluster](/content/mdb-atlas-setup/global-cluster.png)
+
+They are designed for a minimum cluster tier of M30 or greater sharded cluster. For the purpose of this guide we will leave this unselected and move forward. If global clusters fit your use case, you can read more about them and how to configure in the official [MongoDB documentation](https://docs.atlas.mongodb.com/global-clusters/).
```

Comment:
Might be worth briefly mentioning what M30 means.

Reply:
Good shout, I will add that